### PR TITLE
CI/CD Setup

### PR DESCRIPTION
**Pull request by otsembo**
[Setup CI/CD](https://github.com/Moringa-Learning/attendance_moringa/issues/4)

Description

### CODE REVIEW

**Comment by otsembo** [comment]
@layersony This PR is ready for review

File: .github/workflows/flask_ci.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Set up PostgreSQL
+        run: |
+          sudo apt-get install -y libpq-dev
+          export DATABASE_URL="postgresql+psycopg2://postgres:postgres@localhost:5432/test_db"
+          export SECRET_KEY="your_secret_key"
```
Comment (layersony) [inline .github/workflows/flask_ci.yaml]: Could set Environment variables, I have set them up

DATABASE_URL as `SQLALCHEMY_DATABASE_URI` 
SECRET_KEY as `SECRET_KEY`,

File: .github/workflows/flask_ci.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+
+  test:
+    runs-on: ubuntu-latest
+    needs: lint
+
+    services:
+      postgres:
+        image: postgres:13
+        env:
```
Comment (layersony) [inline .github/workflows/flask_ci.yaml:54]: Could set Environment variables, I have set them up
POSTGRES_DB as POSTGRES_DB
POSTGRES_USER as POSTGRES_USER
POSTGRES_PASSWORD as POSTGRES_PASSWORD

**Review by layersony — CHANGES_REQUESTED** [review]

**Comment by otsembo** [comment]
@layersony Added credentials to use CI secrets.

**Pull request merged**